teamwork/apps/profiles/views/BaseView.py

```diff
@@ -9,16 +9,47 @@
 # View Imports
 from teamwork.apps.profiles.views.EditProfileView import edit_profile
 
+def find_available_username(username):
+    """
+    checks if the passed username is available to be used.
+    If not, then return the 'username + #' available
```

Comment:
Now may be a good time to implement better documentation, especially with some simple stuff

i.e
:param str username: User's name.

Reply:
Will do. I'm not familiar w/ the most popular styling for this in python. I used this for reference:
[Google Style Python Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)